themes/openy_themes/openy_lily/openy_lily.install

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @file
+ * Install and uninstall functions for the Open Y Lily Theme.
+ */
+
+/**
+ * Enable openy_block_branch_contacts_info module.
+ */
+function openy_lily_update_8001() {
+    \Drupal::service('module_installer')->install(['jquery_ui_tabs']);
+  }
+  
```

Comment:
Line indentation issues.
No newline at end of file.

Reply:
Fixed in https://github.com/ymcatwincities/openy/pull/2179/commits/b0d332dbd25b89d700d63f8eff66a4d457d46594